### PR TITLE
update:quick-start-with-tidb.md

### PR DESCRIPTION
**Pull request by yanjingfan**
### What is changed, added or deleted? (Required)
修改了quick-start-with-tidb.md to close https://github.com/pingcap/docs-cn/issues/4720

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/issues/4720

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by TomShawn** [comment]
@yanjingfan Thanks for making a contribution! Could you please use a GitHub user account to create a PR and sign the [CLA](https://github.com/pingcap/docs-cn/pull/4742#issuecomment-716954467)?

File: quick-start-with-tidb.md [hunk 1]
```diff
@@ -74,11 +74,21 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
 
 4. 新开启一个 session 以访问 TiDB 数据库。
 
-    {{< copyable "shell-regular" >}}
+    + 首先安装mysql客户端，安装过mysql客户端的则可跳过这一小步骤
```
Comment (kissmydb) [inline quick-start-with-tidb.md:77]:
```suggestion
    + 首先安装 MySQL 客户端，安装过 MySQL 客户端的则可跳过这一小步骤
```

File: quick-start-with-tidb.md [hunk 2]
```diff
@@ -74,11 +74,21 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
...
-    ```shell
-    mysql --host 127.0.0.1 --port 4000 -u root
-    ```
+      {{< copyable "shell-regular" >}}
+
+      ```shell
+      yum -y install mysql
+      ```
+
+    + 使用mysql客户端连接TiDB
```
Comment (kissmydb) [inline quick-start-with-tidb.md:85]:
```suggestion
    + 使用 MySQL 客户端连接 TiDB
```

File: quick-start-with-tidb.md [hunk 3]
```diff
@@ -270,6 +280,14 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
 
 8. 访问集群：
 
+    + 安装mysql客户端，安装过mysql客户端的则可跳过这一小步骤
```
Comment (kissmydb) [inline quick-start-with-tidb.md:283]:
```suggestion
    + 安装 MySQL 客户端，安装过 MySQL 客户端的则可跳过这一小步骤
```

**Review by kissmydb — CHANGES_REQUESTED** [review]

**Comment by TomShawn** [comment]
@yanjingfan This PR is migrated to https://github.com/pingcap/docs-cn/pull/4943.

**Pull request closed without merging**